qiskit_aqua/quantum_instance.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, backend, shots=1024, max_credits=10, seed=None,
             noise_model (qiskit.provider.aer.noise.noise_model.NoiseModel): noise model for simulator
             timeout (float or None): seconds to wait for job. If None, wait indefinitely.
             wait (float): seconds between queries to result
+            cache_config (dict): A dictionary with kwargs to init an instance of CircuitCache
```

Comment:
should be `circuit_cache`? and type should be `CircuitCache`?

Reply:
Updated